src/CommunityToolkit.Maui/Views/Toast/Toast.ios.macos.cs

```diff
@@ -4,15 +4,15 @@
 using Foundation;
 using UIKit;
 
-namespace CommunityToolkit.Maui.Views.Popup.Snackbar;
+namespace CommunityToolkit.Maui.Views.Toast;
 
-class NativeToast : NativePopup
+class Toast : Popup.Popup
```

Comment:
AppleToast

Reply:
My preference to keep it as `Toast` in case we decide to add a `Toast` control to Maui Community Toolkit in the future. 

Then, similar to the Snackbar architecture, we'd have `Toast.shared.cs` `Toast.ios.macos.cs`, `Toast.android.cs`, and `Toast.windows.cs`.

The filename is already `Toast.ios.macos.cs` which means it can only run on iOS + MacCatalyst anyways.